src/components/Button.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Button = (name) => (
+  <button type="button">{name.btnName}</button>
+);
+
```

Comment:
You need to add `prop-types` for the Button component to validate the checking
![image](https://user-images.githubusercontent.com/31897434/72972685-2fa22680-3dcc-11ea-97af-f5fbdeabb70b.png)
https://reactjs.org/docs/typechecking-with-proptypes.html